src/components/Navbar/MenuItem.jsx

```diff
@@ -0,0 +1,3 @@
+export const MenuItem = ({ children }) => {
+  return { children };
```

Comment:
What is the point of creating a **component** that the only thing it does is, display the same element it receives? It is the same as displaying it on the parent level but longer.